tikv-client/src/main/java/com/pingcap/tikv/columnar/TiColumnarChunk.java

```diff
@@ -0,0 +1,23 @@
+package com.pingcap.tikv.columnar;
+
+public class TiColumnarChunk {
```

Comment:
Naming. Please be consistent about you want to use name "ColumnarChunk" or just "Chunk". I've mentioned this for like a million times.

Reply:
DONE